node_modules/oae-core/editprofile/css/editprofile.css

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#editprofile-plea {
+    font-size: 14px;
+    font-style: normal;
+    margin-bottom: 15px;
+}
+
+#editprofile-email-verification .alert {
+    -webkit-align-items: center;
+        -ms-align-items: center;
+            align-items: center;
+    display: -webkit-flex;
+    display:  -ms-flexbox;
+    display:         flex;
+    margin-bottom: 5px;
+}
+
+#editprofile-email-verification .alert > div {
+    /* IE10 requires this to be set explicitly, but it's the default for all other flex-supporting
+       browsers */
+    -ms-flex: 0 1 auto;
+}
+
+#editprofile-email-verification .editprofile-email-verification-content {
+    width: 100%;
+}
+
+#editprofile-email-verification .editprofile-email-verification-content .editprofile-email-verification-content-email {
+    font-weight: bold;
+    word-wrap: break-word;
+}
+
+#editprofile-email-verification .editprofile-email-verification-action {
+    -webkit-flex: 0 0 150px;
+        -ms-flex: 0 0 150px;
+            flex: 0 0 150px;
+    margin-left: 15px;
+}
+
+@media (max-width: 479px) {
+    #editprofile-email-verification .alert {
```

Comment:
We should be able to trim down this CSS considerably by using standard bootstrap, floats. etc.